getting-started/gradle.properties

```diff
@@ -0,0 +1 @@
+quarkusVersion = 999-SNAPSHOT
```

Comment:
as per the discussion in #138, IMHO this is a bad idea, and should be `0.13.0` instead, just like in the Maven POM.

Reply:
It's for the development branch. It will be replaced automatically when releasing.

So for the next release, it will be automatically replaced by `0.14.0` when merged to master.

Reply:
Oh, I've only just now figured that on the `development` branch it's `999-SNAPSHOT` even in the POMs, and the master branch has the released versions.. right, then it makes perfect sense.